src/Components/Projects/Projects.tsx

```diff
@@ -367,11 +367,12 @@ const Projects: React.FC = () => {
           </Modal>
           {/************************* * //delete modal*************** */}
           {/* pagination */}
+          {!isLoading ? (
```

Comment:
you can write it as follows:
`{!isLoading  && <CustomPagination/>}`

This is better for readability, if `isLoading` is falsy it will return nothing , and if it is true, it will return the CustomPagination Component